import/TimeSlider.qml

```diff
@@ -0,0 +1,569 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+/*!
+    \qmltype TimeSlider
+    \inqmlmodule Esri.ArcGISRuntime.Toolkit
+    \since Esri.ArcGISRutime 100.9
+    \brief The slider provides a user interface for manually setting or animating
+    changes to the current time extent of the \c GeoView.
+    A time slider can be bound to a geoView
```

Comment:
An open line follows \brief and additional content follows that one open line.  This convention is throughout the qt_cpp and qt_qml apis. 
```suggestion
    
    A time slider can be bound to a geoView
```